README.md

```diff
@@ -1,5 +1,5 @@
-# Learnspigot Discord bot
-The discord bot for the [Learnspigot Discord server](https://learnspigot.com/discord). This powers everything from verification to reputation in help forums and checking the stock of WIX.  
+# LearnSpigot Discord Bot
```

Comment:
This should be in its own pr. Try to not add multiple features in one pr

Reply:
It is indeed tested